clients/vault/src/requests/structs.rs

```diff
@@ -199,11 +199,26 @@ impl Request {
 				)
 			},
 			|result| async {
-				match result {
-					Ok(ok) => Ok(ok),
-					Err(err) if err.is_rpc_disconnect_error() =>
-						Err(runtime::RetryPolicy::Throw(err)),
-					Err(err) => Err(runtime::RetryPolicy::Skip(err)),
+				match result.map_err(Into::<EnrichedError>::into) {
+					Ok(te) => Ok(te),
```

Comment:
What does `te` mean by the way?

Reply:
TBH I don't know. It was already [defined](https://github.com/pendulum-chain/spacewalk/pull/453/files#diff-67810aa5914ac045b15035e36b5de374c73cabd4d44ae06f760cfc3dd23c802eL235)  like `te` before.

Reply:
haha yes but in the other [call to notify_retry](https://github.com/pendulum-chain/spacewalk/blob/main/clients/runtime/src/rpc.rs#L235) was like that. Don't worry I will put both back to `ok`.

Reply:
Ah that's what you meant, sorry I didn't see the other one haha.